templates/taxi/manufacturer_list.html

```diff
@@ -1,15 +1,16 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+    <h1>Manufacturer List <a style="float: right;" href="{% url 'taxi:manufacturer-create' %}">+</a></h1>
```

Comment:
Ensure that the URL pattern `taxi:manufacturer-create` is correctly defined in your `urls.py` file, as it is used here to link to the manufacturer creation page.